DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/_CourseDetailsExpandable.cshtml

```diff
@@ -67,7 +67,10 @@
 
     </dl>
 
-    <a class="nhsuk-button nhsuk-u-margin-right-2" role="button" href="" aria-label="Edit course details">
+    <a class="nhsuk-button nhsuk-u-margin-right-2" role="button" aria-label="Edit course details"

```

Comment:
Should probably technically drop the other 2 attributes here onto their own lines

Reply:
changed